common/flatpak-run.c

```diff
@@ -316,6 +316,106 @@ flatpak_run_add_pcsc_args (FlatpakBwrap *bwrap)
   flatpak_bwrap_set_env (bwrap, "PCSCLITE_CSOCK_NAME", sandbox_pcsc_socket, TRUE);
 }
 
+static gboolean
+flatpak_run_cups_check_server_is_socket (const char *server)
+{
+  if (g_str_has_prefix (server, "/") && !g_strv_contains ((const char *const*)server, ":"))
+    return TRUE;
+
+  return FALSE;
+}
+
+/* Try to find a default server from a cups confguration file */
+static char *
+flatpak_run_get_cups_server_name_config (const char *path)
+{
+  g_autoptr(GFile) file = g_file_new_for_path (path);
+  g_autoptr(GError) my_error = NULL;
+  g_autoptr(GFileInputStream) input_stream = NULL;
+  g_autoptr(GDataInputStream) data_stream = NULL;
+  size_t len;
+
+  input_stream = g_file_read (file, NULL, &my_error);
+  if (my_error)
+    {
+      g_warning ("CUPS configuration file '%s': %s", path, my_error->message);
```

Comment:
I think by default there is no client.conf, so please don't warn here. Maybe g_debug.